proto/redpanda/core/admin/v2/datalake.proto

```diff
@@ -0,0 +1,83 @@
+// Copyright 2025 Redpanda Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package redpanda.core.admin.v2;
+
+import "proto/redpanda/core/pbgen/options.proto";
+import "proto/redpanda/core/pbgen/rpc.proto";
+
+option (redpanda.core.pbgen.cpp_namespace) = "proto::admin";
+
+service DatalakeService {
+    // Gets the coordinator state for the given topics.
+    rpc CoordinatorGetState(CoordinatorGetStateRequest)
+        returns (CoordinatorGetStateResponse) {
+        option (redpanda.core.pbgen.rpc) = {
+            authz: SUPERUSER
+        };
+    }
+}
+
+message CoordinatorGetStateRequest {
+    // Names of topics whose state to return. If empty, returns state for all
+    // topics.
+    repeated string topics_filter = 1;
+}
+
+message CoordinatorGetStateResponse {
```

Comment:
A GetCoordinatorStateResponse should return a single field of a message type CoordinatorState